src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+
+    private static final double PI = 3.14;
+
+    public Circle(Colors color, double side) {
```

Comment:
The parameter name 'side' for the Circle constructor is misleading, as a circle does not have sides. A more appropriate name would be 'radius'.